docs/topics/settings.rst

```diff
@@ -851,6 +851,13 @@ FEED_STORAGE_GCS_ACL
 The Access Control List (ACL) used when storing items to :ref:`Google Cloud Storage <topics-feed-storage-gcs>`.
 For more information on how to set this value, please refer to the column *JSON API* in `Google Cloud documentation <https://cloud.google.com/storage/docs/access-control/lists>`_.
 
+.. setting:: FEED_STORAGE_TIMEOUT
+
+FEED_STORAGE_TIMEOUT
+--------------------
+The Feed Storage Timeout you can configure the timeout of feed upload oprations.
+Currently implemented only for Google Cloud Storage.
```

Comment:
I think we should also mention the value unit (seconds?), as well as the default value.